custom-style-helper.js

```diff
@@ -0,0 +1,13 @@
+import '@webcomponents/shadycss/custom-style-interface.min.js';
+
+export const addCustomStyle = (style) => {
+	const elem = document.createElement('style');
+	elem.textContent = style;
+	const head = document.getElementsByTagName('head')[0];
+	head.appendChild(elem);
```

Comment:
This could actually be done inside of `getStyle` below.  Not sure if it makes any difference.  That's how it was done in their sample: https://github.com/BrightspaceUI/core/pull/46#issuecomment-483284483.